File: CS/Data/DepartmentContextInitializer .cs
using System;
using System.Collections.Generic;
using System.Data.Entity;

namespace Scaffolding.ValidationErrors.Model {
    public class DepartmentContextInitializer : DropCreateDatabaseIfModelChanges<DepartmentContext> {
        protected override void Seed(DepartmentContext context) {
            base.Seed(context);

            context.Departments.Add(new Department { Budget = 138, Name = "Music", StartDate = new DateTime(1988, 9, 14) });
            context.Departments.Add(new Department { Budget = 572, Name = "Journalism", StartDate = new DateTime(1994, 5, 23) });
            context.Departments.Add(new Department { Budget = 349, Name = "Management", StartDate = new DateTime(1994, 3, 7) });

            context.SaveChanges();
        }
    }
}

File: VB/Scaffolding.ValidationErrors/Data/DepartmentContextInitializer .vb
Imports System
Imports System.Collections.Generic
Imports System.Data.Entity

Namespace Scaffolding.ValidationErrors.Model
	Public Class DepartmentContextInitializer
		Inherits DropCreateDatabaseIfModelChanges(Of DepartmentContext)

		Protected Overrides Sub Seed(ByVal context As DepartmentContext)
			MyBase.Seed(context)

			context.Departments.Add(New Department With {.Budget = 138, .Name = "Music", .StartDate = New Date(1988, 9, 14)})
			context.Departments.Add(New Department With {.Budget = 572, .Name = "Journalism", .StartDate = New Date(1994, 5, 23)})
			context.Departments.Add(New Department With {.Budget = 349, .Name = "Management", .StartDate = New Date(1994, 3, 7)})

			context.SaveChanges()
		End Sub
	End Class
End Namespace